x-pack/legacy/plugins/siem/public/components/charts/chart_holder.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiFlexItem, EuiText, EuiFlexGroup } from '@elastic/eui';
+import styled from 'styled-components';
+import { ChartSeriesData, checkIfAllValuesAreZero } from './common';
+import * as i18n from './translation';
+
+const FlexGroup = styled(EuiFlexGroup)<{ height?: string | null; width?: string | null }>`
+  height: ${({ height }) => (height ? height : '100%')};
+  width: ${({ width }) => (width ? width : '100%')};
+  position: relative;
+  margin: 0;
+`;
+
+FlexGroup.displayName = 'FlexGroup';
+
+export const ChartHolder = ({
```

Comment:
wondering if we can rename this to be more descriptive? I would expect a chart to be inside it, but instead it shows the error messaging when there is no data or zero data. maybe `ChartPlaceholder`?